powerlevel9k.zsh-theme

```diff
@@ -718,21 +718,78 @@ prompt_command_execution_time() {
 # Dir: current working directory
 set_default POWERLEVEL9K_DIR_PATH_SEPARATOR "/"
 set_default POWERLEVEL9K_HOME_FOLDER_ABBREVIATION "~"
-set_default POWERLEVEL9K_DIR_SHOW_WRITABLE false
```

Comment:
I see you've removed this field, but it's still in use in the code? Was it an accidental delete?